app/views/media_objects/_item_view.html.erb

```diff
@@ -87,6 +87,13 @@ Unless required by applicable law or agreed to in writing, software distributed
 
   <% if @currentStream.present? and @currentStream.derivatives.present? %>
   <script>
+    function advancePlaylist() {
+      nextUrl = $('a.playable:not(a.current-stream):first').attr('href')
```

Comment:
Won't this find the first playable stream that's not currently playing? e.g., if the first stream is current, it'll find the second one; otherwise, it'll find the first one.